src/autopas/containers/CellBlock3D.h

```diff
@@ -34,11 +34,12 @@ class CellBlock3D : public CellBorderAndFlagManager {
    * @param vec vector of ParticleCells that this class manages
    * @param bMin lower corner of the cellblock
    * @param bMax higher corner of the cellblock
-   * @param interactionLength minimal size of each cell
+   * @param interactionLength max. radius of interaction between particles
+   * @param cellSize cell size ralative to interactionLength
```

Comment:
```suggestion
   * @param cellSize cell size relative to interactionLength
```

Reply:
I'll change the name from cellSize to cellSizeFactor to show that this is a relative value.
> Additionally, this factor should always be bigger than 1.0, as a decreased cell size is normally not allowed.
>
Regarding this PR, this is correct, but support for smaller cell sizes is coming soon (see branch linked-cell-smaller-cell-sizes).